Cargo.toml

```
@@ -20,7 +20,7 @@ async-tokio = ["gpio-cdev/async-tokio"]
 default = [ "gpio_cdev", "gpio_sysfs" ]
 
 [dependencies]
-embedded-hal = "=1.0.0-alpha.4"
+embedded-hal = { path = "../embedded-hal" }
```

This needs to be undone before the merge

---

```suggestion
embedded-hal = { git = "https://github.com/rust-embedded/embedded-hal" }
```
So that we can run the CI here.

---

Done. Unfortunately this can't build at the moment since the [try_* prefix was recently removed](https://github.com/rust-embedded/embedded-hal/commit/98196625a02dde104197b51c003b08d96fea3207), and this implementation's methods are not compatible with this yet.